src/Ozone_Monitor/parm/OznMon_config

```diff
@@ -9,217 +9,76 @@
-
-      #rstprod data is not allowed on S4
-      CHGRP_CMD=echo
-   elif [[ $MY_MACHINE = "jet" ]]; then
-
-      module load grads/2.2.1
-      export GRADS=`which grads`
-      export STNMAP=`which stnmap`
-      module use /lfs4/HFIP/hfv3gfs/nwprod/hpc-stack/libs/modulefiles/stack
-      module load hpc/1.1.0 hpc-intel/18.0.5.274 prod_util/1.2.2
-
-      export SUB=/apps/slurm/default/bin/sbatch
-      export COMPRESS=${COMPRESS:-gzip}
-      export UNCOMPRESS=${UNCOMPRESS:-"gunzip -f"}
-
-      export PARTITION_OZNMON=${PARTITION_OZNMON:-vjet}
-
-   elif [[ $MY_MACHINE = "wcoss2" ]]; then
-      export COMPRESS=${COMPRESS:-gzip}
-      export UNCOMPRESS=${UNCOMPRESS:-"gunzip -f"}
-   fi
-  
+#-------------------------------------------------------
+#  Source Mon_config for GSI-monitor definitions common
+#  to all monitors.
+#
+readonly dir_root=$(cd "$(dirname "$(readlink -f -n "${BASH_SOURCE[0]}" )" )/.." && pwd -P)
+
+MON_CONFIG=$dir_root/../../parm/Mon_config
+source $MON_CONFIG
+
+export RUN=${RUN:-gdas}
+
+export WEB_DIR=${WEB_DIR:-/home/people/emc/www/htdocs/gmb/gdas/es_ozn/pngs}
+
+
+#-------------------------------------------------------------------
+#  Note:  The file header in the oznstat file has slightly changed.
+#         The default setting is to use that new header format, but
+#         at the moment there is no operational source doing so.  So
+#         setting this to 0 will make all file header reads assum the 
+#         older format (which does work with the new file header). 
+#         Once sources begin to use the new format this can either be
+#         set to 1 or may go away.
+#
+export OZNMON_NEW_HDR=${OZNMON_NEW_HDR:-0}
+
+
+export MY_OZNMON=${MY_GSI_MONITOR}/src/Ozone_Monitor
+export OZN_DE_SCRIPTS=${OZN_DE_SCRIPTS:-${MY_OZNMON}/data_xtrct/ush}
+
+export OZN_DE_SORC=${OZN_DE_SORC:-${MY_OZNMON}/data_xtrct/sorc}
+export OZN_DE_EXEC=${OZN_DE_EXEC:-${MY_OZNMON}/data_xtrct/exec}
+export OZN_IG_SCRIPTS=${OZN_IG_SCRIPTS:-${MY_OZNMON}/image_gen/ush}
+export OZN_IG_GSCRPTS=${OZN_IG_GSCRPTS:-${MY_OZNMON}/image_gen/gscripts}
+export OZN_EXEDIR=${OZN_EXEDIR:-${MY_OZNMON}/exec}
+
+export HOMEgdas_ozn=${HOMEgdas_ozn:-${MY_OZNMON}/nwprod/gdas_oznmon}
+export FIXgdas_ozn=${FIXgdas_ozn:-${HOMEgdas_ozn}/fix}
+export HOMEoznmon=${HOMEoznmon:-${MY_OZNMON}/nwprod/oznmon_shared}
+
+#
+#  Add these gfs definitions to accomodate the changes
+#  to the J-Job from the fv3_impl merge
+#
+export PARMgfs_ozn=${PARMgfs_ozn:-${HOMEgdas_ozn}/parm}
+export SCRgfs_ozn=${SCRgfs_ozn:-${HOMEgdas_ozn}/scripts}
+export FIXgfs_ozn=${FIXgfs_ozn:-${HOMEgdas_ozn}/fix}
+
+#
+#  The OZN_TANKDIR will be the location for the extracted data files and
+#  the control files used for image plotting.  This is the base definition
+#  and the succedding scripts will construct and use subdirectories under
+#  this location.
+
+if [[ $GLB_AREA -eq 1 ]]; then
+   OZN_TANKimg=${TANKDIR}/imgn/${OZNMON_SUFFIX}/${RUN}
+   NEWtank=${TANKDIR}/stats/${OZNMON_SUFFIX}
```

Comment:
I believe this should be `NEWtank=${TANKDIR}/oznmon/stats/${OZNMON_SUFFIX}`.

Reply:
And OZN_TANKimg probably needs `/oznmon/` affixed as well.

Reply:
Maybe, but this is part of a larger discussion.  Ops and developers don't use the same storage scheme for extracted monitor data.   Further complicating things, the data locations have changed with every major release.

Currently gfs v16 in ops uses gfs (suffix)/gdas|gfs.yyyymmdd/hh/atmos/monitor to store data.  Internally the monitors use a very similar formula, though without the atmos.  So it's nbns/stats/suffix/gdas|gfs.yyyymmdd/hh/monitor.

Developers running their workflows in rocoto are now commonly using monitor/stats/suffix, as you suggest, though this can vary and does.

For a while I tried to keep all of the monitors in sync with ops and developers.  It proved quite difficult.  

For images the internal storage is nbns/imgn/suffix/gdas|gfs/monitor/pngs.  The image generation portion of the monitors isn't used in ops or global-workflow, so it hasn't changed in quite some time.  

Initial discussions about the next generation of monitoring have recently begun.  The idea is to sunset the current monitors and replace them with a method that is more modular, likely a yaml driven python solution, which would be more adaptable to the several extracted data storage schemes.

So all of that is a long way around saying that the `$NEWtank ` definition, while inconsistent with some workflows, is as intended.   I hope that made sense.

Reply:
@DavidHuber-NOAA After re-reading my explanation for the storage scheme I don't like it and I think it should be more user-friendly.  I'm going to see if I can't accommodate both storage schemes, probably with a switch in the RadMon_user_settings file.  I'll incorporate that change in this issue and will close this PR.  

As always I appreciate your thoughtful reviews.  Thanks.